src/components/EventEdit.tsx

```diff
@@ -55,7 +57,7 @@ class EventEdit extends React.PureComponent<PropsType> {
     location: string;
     description: string;
     journalUid: string;
-
```

Comment:
This space was put there on purpose. I don't blame you for now realising it, because it's terribly hacky what's going on there, but the idea is that the above are actual properties of the event, and below are just local state.

So please keep the space and put the rrule options (why not just call it rrule?) below timezone.

Reply:
ill change it to just rrule

Reply:
rrule need to be before the space, not after. More specifically, I'd probably put it even before journalUid which I think is also not really part of the core event state.